README.md

```diff
@@ -11,16 +11,16 @@
 
 cder (_see-der_) is a database seeding tool to help you import fixure data in your local environment.
 
-Generating seeds programmatically is a easy task, but maintaining them is not.
+Generating seeds programmatically is an easy task, but maintaining them is not.
 Everytime when your schema is changed, your seed can be broken.
 It costs your team extra efforts to keep them updated.
 
 #### with cder you can:
 - maintain your data in a readable format, separated from the seeding program
 - handle reference integrities on-the-fly, using **embedded tags**
-- reuse existing struct and insert function, with only a little glue code is needed
+- reuse existing structs and insert function, with only a little glue code is needed
```

Comment:
```suggestion
- reuse existing structs and insert function,s with only a little glue code is needed
```

Reply:
@kenkoooo  perhaps you meant `s` comes into before comma, right?

```suggestion
- reuse existing structs and insert functions, with only a little glue code is needed
```

Reply:
Thanks! Good catch!